src/__tests__/sheet-test.tsx

```diff
@@ -43,9 +43,10 @@ test('Sheet', async () => {
 
     const closeButton = await within(sheet).findByRole('button', {name: 'Close'});
 
-    await userEvent.click(closeButton);
-    await waitForElementToBeRemoved(sheet);
-}, 30000);
+    await act(() => userEvent.click(closeButton));
```

Comment:
:man_shrugging:

Reply:
why? afaik RTL already wraps these click events with act internally

Reply:
I'm not sure. But without this I was getting missing act errors.